string FrameID
string MAC
---
miflora_msgs/miflora sensor